### ISSUE TITLE
[Accessibility]: Default search context read letter-by-letter

### ISSUE THREAD

**Issue by limitedmage** [sourcegraph/sourcegraph#43132]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34188

### Problem description

The badge for the default search context in the search context dropdown is being read letter-by-letter (D-E-F-A-U-L-T) instead of as a single word.

<img width="534" alt="image" src="https://user-images.githubusercontent.com/206864/196555579-8ee55656-61c2-440b-a9cc-515bf5d15e79.png">

### Expected behavior

It should be read as a single word.

### Additional details

This is probably happening because the text is uppercase in the HTML. It should instead be sentence-case in the HTML and use CSS `text-transform` to make it uppercase.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by GitStart. I have assigned the label 'gitstart' and added the assignee 'gitstart-sourcegraph'.

/cc @lguychard
